project-progression-template.md

```diff
@@ -4,24 +4,47 @@
+5.  Does your project currently have a CLA? If so, please link to the documentation of your CLA.(Using the Developer Certificate of Origin is considered a CLA.)
+6.  Have you read the Consortium’s project acceptance process and understand the IP and trademark statuses of project’s donated to the Consortium? [LINK] (A “no” answer is okay; the committee can answer questions about the policy.)
+7.  Does the project currently have a code of conduct? (If so, please specify and or link to code of conduct page in your documentation)
+8.  Does the project currently have a vulnerability disclosure contact and process? (If so, please link)
+9.  Are any government entities currently providing funding that supports this project? If so, please list nature of funding.
+### Project Functionality
+10.  What does this project do?
+- 280 char summary:
+- Extended answer:
+11. What proprietary dependencies does this project have, if any? For example, it is an open source library that is used on a proprietary platform.
+### Project Fit
+12. What problem is this project solving, and how does that advance the CCC’s mission? For reference, the mission statement is: Confidential Computing is the protection of data in use by performing computation in a hardware-based Trusted Execution Environment. [Mission Statement](README.md)
+- 280 char summary:
+- Extended answer:
+13. Is the project currently compatible with, or does it depend upon or integrate with, existing CCC projects? (Note this is not a requirement and does not impact acceptance; this is just for information gathering) 
+14. Is the project used in production environments today that you are aware of?
+15. Are there currently any commercial offerings of this project?
+### Current contributors and contribution structures
+16. What companies or groups are currently contributing to this project? Do you have an approximate breakdown of contribution percentage by company/group affiliation over the past 6 months?
+17. What incentive or motivation is there for someone to be an external contributor to this project?
+- 280 char summary:
+- Extended answer:
+### Project Goals
+18. Why was this project open sourced (or intended to be open sourced)?
+- 280 char summary:
+- Extended answer:
+19. What are the project’s goals as an open source project?
+- 280 char summary:
+- Extended answer:
+20. How does donation to the CCC align with the project’s goals?
+- 280 char summary:
+- Extended answer:
+21. What outcome does the project leadership anticipate as a result of donation to the CCC?
+- 280 char summary:
+- Extended answer:
+22. What efforts, if any, have already been applied or are currently in progress to achieve those goals?
+23. What would the project leadership like to see from the CCC leadership to support achieving those goals?
+24. After being donated, what level of involvement will the donors have with the project? Can the project’s current developers commit to ongoing leadership of the open source project?
```

Comment:
This section is important for understanding *why* the projects are joining the Consortium so the TAC and Outreach Committees can understand how to support them (or if they aren't being donated for reasons that are going to support good outcomes -- ie "no one is maintaining this anymore and we're also not going put any effort behind it anymore so here's some source code"--that's not a likely outcome for success). Particularly for the Outreach Committee, the Committee is there to support the health and growth of projects, but we have no information like this about the three that have already been donated.

Reply:
There's another way of looking at this question, which is "what is the community health for the project"?  Not all projects will be sourced in a company and then donated, and the phrase "donors" feels a little odd to me in this context.  I'd like to ask more about maintainers and contributors.